data/names.txt

```diff
@@ -216,6 +224,17 @@ phrase "singular nouns"
 		"Banner"
 		"Voyage"
 		"Rose"
+		"Song"
+		"Hawk"
+		"Raven"
```

Comment:
I've generally tried to avoid having ship model names appear in part of the ship names, to avoid confusion. (Although, it looks like there's a "Hawk" and "Falcon" in the pirate names too.)